c++/src/capnp/rpc.c++

```diff
@@ -4655,6 +4742,18 @@ private:
 
     // No more using `call` after this point, as it now belongs to the context.
 
+    if (call.getIsRealtime()) {
```

Comment:
It's a little jarring seeing the comment just above this saying that `call` is no longer safe to use... and then seeing this line use it.

I guess technically the `RpcCallContext` won't have destroyed `message` at this point... but maybe better to use `hints.isRealtime` instead to avoid any confusion.